src/main/resources/assets/world-host/lang/de_de.json

```
@@ -0,0 +1,48 @@
+{
+  "world-host.config.title": "World Host Config",
+  "world-host.config.serverIp": "Server URI",
+  "world-host.config.showOnlineStatus": "Online-Status anzeigen",
+  "world-host.config.showOnlineStatus.tooltip": "Online-Status auf dem Titelbildschirm anzeigen",
+  "world-host.config.enableFriends": "Aktiviere Freunde",
+  "world-host.config.enableFriends.tooltip": "Aktiviert alle auf Freunde bezogenen GUIs und das gesamte Freundesystem.\nWenn du es deaktivierst und wieder aktivierst, sind alle Ihre Freunde noch da.",
```

This should probably be less formal to fit in with all the other strings.
```suggestion
  "world-host.config.enableFriends.tooltip": "Aktiviert alle auf Freunde bezogenen GUIs und das gesamte Freundesystem.\nWenn du es deaktivierst und wieder aktivierst, sind alle deine Freunde noch da.",
```

---

Oh, I thought I replaced it in all places, sorry about that